examples/inverse/evoked_ers_source_power.py

```diff
@@ -17,7 +17,7 @@
 # License: BSD-3-Clause
 
 # %%
-import os.path as op
+from pathlib import Path
```

Comment:
```suggestion
```

don't need to import it in this case, since you're not constructing any Pathlib objects yourself (just using ones MNE-Python made for you)